src/main/scala/ai/privado/exporter/ConsoleExporter.scala

```diff
@@ -26,14 +27,10 @@ object ConsoleExporter {
     val uniqueThirdParties = mutable.Set[String]()
 
     // SourceId - Name Map
-    sources.foreach(source => {
-      sourceNameIdMap.addOne(
-        source.id.replaceAll("\"", "") -> source.name.replaceAll("\"", "")
-      )
-    })
+    sourceNameIdMap = sources.map((source) => (source.id, source.name)).toMap
```

Comment:
Instead of declaring it as `var` we can say `val sourceNameIdMap = sources.map((source) => (source.id, source.name)).toMap`

Similar to this we can apply to leakageSourceMap, processSourceMap, collectionsSourceMap, uniqueThirdParties